### PR TITLE
fix: handle exception cases while no plugins

### PR DESCRIPTION
**Pull request by shanshandev**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/feflow?pullRequest=142) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/feflow?pullRequest=142) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: fXy-during<br/>:white_check_mark: cpselvis<br/>:x: salomezhang<br/><hr/>**salomezhang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/feflow?pullRequest=142) it.</sub>

**Comment by cpselvis** [comment]
image

这个需要配置正确的 github 用户名信息

File: packages/feflow-cli/src/core/native/list.ts [hunk 1]
```diff
@@ -32,21 +32,39 @@ function loadModuleList(ctx: any) {
 module.exports = (ctx: any) => {
     ctx.commander.register('list', 'Show all plugins installed.', () => {
         const list = loadModuleList(ctx);
+        let flag = 0;
 
         console.log('You can search more templates or plugins through https://feflowjs.com/encology/');
         console.log('===============================================');
+        if (!list.length) {
+            console.log(chalk.magenta('暂未安装任何模板和插件'));
```
Comment (cpselvis) [inline packages/feflow-cli/src/core/native/list.ts]: 这个地方统一改成英文

File: packages/feflow-cli/src/core/native/list.ts [hunk 2]
```diff
@@ -32,21 +32,39 @@ function loadModuleList(ctx: any) {
...
         console.log('plugins');
         list.map(function (name) {
-            if (!/^feflow-plugin-|^@[^/]+\/feflow-plugin-/.test(name)) {
+            if (/^feflow-plugin-|^@[^/]+\/feflow-plugin-/.test(name)) {
                 console.log(chalk.magenta(name));
+                flag = 1;
             }
         });
+        if (!flag) {
+            console.log(chalk.magenta('暂未安装任何插件'));
```
Comment (cpselvis) [inline packages/feflow-cli/src/core/native/list.ts]: 这个地方统一改成英文

File: packages/feflow-cli/src/core/native/list.ts [hunk 3]
```diff
@@ -32,21 +32,39 @@ function loadModuleList(ctx: any) {
...
                 console.log(chalk.magenta(name));
+                flag = 1;
             }
         });
+        if (!flag) {
+            console.log(chalk.magenta('暂未安装任何模板'));
+        }
+
+        flag = 0;
+        // print plugins list
```
Comment (cpselvis) [inline packages/feflow-cli/src/core/native/list.ts]: 这行注释意义不大可以删掉

File: packages/feflow-cli/src/core/native/list.ts [hunk 4]
```diff
@@ -32,21 +32,39 @@ function loadModuleList(ctx: any) {
 module.exports = (ctx: any) => {
     ctx.commander.register('list', 'Show all plugins installed.', () => {
         const list = loadModuleList(ctx);
+        let flag = 0;
```
Comment (cpselvis) [inline packages/feflow-cli/src/core/native/list.ts]: 这个 flag 命名需要调整，可以改成 templateCnt 和 pluginCnt。不要一个 flag 用多次

**Pull request merged**